<launch>
  <node name="semantic_map_server_node"  pkg="semantic_map_server" type="semantic_map_server_node">
    <param name="map_file" value="$(find semantic_map_server)/config/semanticmap.pgm" />
    <param name="height" value="512"/>
    <param name="width" value="600"/>
    <param name="resolution" value="0.05"/>
    <param name="origin_x" value="0.0"/>
    <param name="origin_y" value="0.0"/>
  </node>
</launch>
